main.py

```diff
@@ -1,10 +1,59 @@
+import json
+
 import init_django_orm  # noqa: F401
 
-from db.models import Race, Skill, Player, Guild
+from db.models import Race, Guild, Skill, Player # noqa
+
+
+def get_or_create_race(race: dict) -> Race:
+    obj, _ = Race.objects.get_or_create(
+        name=race.get("name"),
+        description=race.get("description")
+    )
+    return obj
+
+
+def get_or_create_guild(guild: dict | None) -> Guild | None:
+    if guild is None:
+        return
```

Comment:
Returning `None` explicitly would be clearer here. Consider using `return None` to make it explicit that the function returns None when the guild is None.